client/my-sites/people/people-list-item/index.jsx

```diff
@@ -1,36 +1,35 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import PureRenderMixin from 'react-pure-render/mixin';
+import React, { PureComponent, PropTypes } from 'react';
+import { connect } from 'react-redux';
 import classNames from 'classnames';
-import omit from 'lodash/omit';
+import { identity } from 'lodash';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import CompactCard from 'components/card/compact';
 import PeopleProfile from 'my-sites/people/people-profile';
-import analytics from 'lib/analytics';
 import config from 'config';
+import { recordGoogleEvent } from 'state/analytics/actions';
 
-export default React.createClass( {
+export class PeopleListItem extends PureComponent {
+	static propTypes = {
+		translate: PropTypes.func,
+	};
 
-	displayName: 'PeopleListItem',
+	static defaultProps = {
+		translate: identity,
+	};
 
-	mixins: [ PureRenderMixin ],
-
-	navigateToUser() {
-		window.scrollTo( 0, 0 );
-		analytics.ga.recordEvent( 'People', 'Clicked User Profile From Team List' );
-	},
-
-	userHasPromoteCapability() {
+	userHasPromoteCapability = () => {
 		const site = this.props.site;
 		return site && site.capabilities && site.capabilities.promote_users;
-	},
+	};
 
-	canLinkToProfile() {
+	canLinkToProfile = () => {
 		const site = this.props.site,
 			user = this.props.user;
```

Comment:
while we're in here let's add a new `const` for the `user` variable too. our style is different for `var` than it is for `let` and `const`